Sources/AppStoreConnectClient/AppStoreConnectClient.swift

```diff
@@ -50,97 +50,141 @@ public struct AppStoreConnectClient {
+            return result
         case .badRequest(let result):
-            return AppStoreConnectError.badRequest(
+            throw AppStoreConnectError.badRequest(
                 errors: AppStoreConnectError.parseErrorDescription(from: try result.body.json)
-                )
+            )
         case .forbidden(let result):
-            return AppStoreConnectError.forbidden(
+            throw AppStoreConnectError.forbidden(
                 errors: AppStoreConnectError.parseErrorDescription(from: try result.body.json)
-                )
+            )
         case .notFound(let result):
-            return AppStoreConnectError.notFound(
+            throw AppStoreConnectError.notFound(
                 errors: AppStoreConnectError.parseErrorDescription(from: try result.body.json)
-                )
+            )
         case .unauthorized(let result):
-            return AppStoreConnectError.unauthorized(
+            throw AppStoreConnectError.unauthorized(
                 errors: AppStoreConnectError.parseErrorDescription(from: try result.body.json)
-                )
+            )
         case .undocumented(let statusCode, _):
-            return AppStoreConnectError.serverError(errorCode: statusCode)
-            
-        default:
-            return AppStoreConnectError.unexpectedError(errors: "\(response)")
+            throw AppStoreConnectError.serverError(errorCode: statusCode)
         }
     }
     
-    /// Handles error responses returned by the App Store Connect API when fetching app collections.
-    /// - Parameter response: The response received from the API.
+    /// Fetches a collection of TestFlight builds for a specified app from the App Store Connect API,
+    /// handles error responses returned by the App Store Connect API when fetching TestFlight builds.
+    /// End-point: v1/builds
+    /// - Parameter app: The app for which to fetch builds.
+    /// - Returns: An array of `Build` objects.
     /// - Throws: An error of type `AppStoreConnectError` if the response indicates an error.
-    private func handleErrors(
-        from response: Operations.apps_hyphen_get_collection.Output
-    ) throws -> AppStoreConnectError {
+    public func fetchBuilds(for app: Application) async throws -> [Build] {
+        let response = try await client.builds_hyphen_get_collection(
+            query: .init(
+                filter_lbrack_app_rbrack_: [app.id],
+                sort: Operations.builds_hyphen_get_collection.Input.Query.sortPayload?.init([._hyphen_version]),
+                fields_lbrack_builds_rbrack_: Operations.builds_hyphen_get_collection.Input.Query.fields_lbrack_builds_rbrack_Payload?.init(
+                    [.version, .uploadedDate, .minOsVersion]
+                )
+            )
+        )
         switch response {
+        case .ok(let okResponse):
+            let result = try okResponse.body.json.data.compactMap { Build(schema: $0) }
```

Comment:
Try remove this constant. 
`return try okResponse.body.json....`

Reply:
Removed.